src/Dfe.EarlyYearsQualification.Web/Models/Content/RatioRequirementModel.cs

```diff
@@ -2,18 +2,18 @@ namespace Dfe.EarlyYearsQualification.Web.Models.Content;
 
 public class RatioRequirementModel
 {
-    public bool ApprovedForLevel2 { get; set; }
+    public QualificationApprovalStatus ApprovedForLevel2 { get; set; }
 
-    public bool ApprovedForLevel3 { get; set; }
+    public QualificationApprovalStatus ApprovedForLevel3 { get; set; }
 
-    public bool ApprovedForLevel6 { get; set; }
+    public QualificationApprovalStatus ApprovedForLevel6 { get; set; }
 
-    public bool ApprovedForUnqualified { get; set; }
+    public QualificationApprovalStatus ApprovedForUnqualified { get; set; }
 
-    public bool IsNotFullAndRelevant
-    {
-        get { return ApprovedForUnqualified && !ApprovedForLevel2 & !ApprovedForLevel3 && !ApprovedForLevel6; }
-    }
+    public bool IsNotFullAndRelevant =>
+        ApprovedForLevel2 != QualificationApprovalStatus.Approved
+        && ApprovedForLevel3 != QualificationApprovalStatus.Approved
+        && ApprovedForLevel6 != QualificationApprovalStatus.Approved;
```

Comment:
If the value is now an enum should we also specify what the acceptable value is here for ApprovedForUnqualified?

Reply:
With every qualification being approved at unqualified, it seems a bit redundant to check it. I can add it is as an Approved check to be more verbose?

Reply:
Nah thinking about it more, leave it as is :-)